### ISSUE TITLE
Change eu_concern_species check source

### ISSUE THREAD

**Issue by SanderDevisscher** [inbo/aspbo#63]
currently I check wheter or not "eu_concern_species.tsv" on the bucket has changed compared to the eu_concern list on gbif. However due to changes in the worklfow eu_concern_species.tsv is no longer updated on the bucket, while "eu_concern_species_processed.RData" is. 

- [x] read "eu_concern_species_processed.RData" from the bucket <sup>1</sup>
- [x] check flow for changes
- [ ] remove "eu_concern_species.tsv" from bucket

<sup>1</sup> @mvarewyck @Yingjie-ZhangOA how can I best read this file from the bucket ?

**Comment by mvarewyck** [comment]
> @mvarewyck @Yingjie-ZhangOA how can I best read this file from the bucket ?

`alienSpecies::loadTabularData(type = "unionlist")`

**Comment by SanderDevisscher** [comment]
thanks

**Comment by SanderDevisscher** [comment]
@TheJenne18 could it be possible I don't have the necessary rights to remove files (objects) from the UAT - bucket?

I use the following code but get a `Client error: (403) Forbidden` error. The file is also still present on the bucket. 

```r
aws.s3::delete_object("eu_concern_species.tsv", bucket = Sys.getenv("UAT_bucket"), region = 'eu-west-1')
``` 

PS I connected to the bucket using `connect_to_bucket(bucket_name = Sys.getenv("UAT_bucket"))` and I was able to retrieve a filelist.
